conversations/v3/configurations.yaml

```diff
@@ -629,12 +627,12 @@ components:
           - callbacks:write
           - callbacks:read
         id:
-          telegramId: 12341234
+          botId: "12341234"
```

Comment:
Could not test it. Is this correct?

Reply:
yes, as we do not have control over the id, modelling as a string is best practice